tests/test_verify.py

```diff
@@ -0,0 +1,33 @@
+import src.verify
+from src.verify import check
+from src.verify import main
+from pytest_mock import MockerFixture
+
+def test_verify():
+    N = (3, 4, 5, 1, 2, 3, 4, 9, 13, 0)
+    assert check(N, 4) == True
+    assert check(N, 10) == False
+
+def test_main_a(mocker: MockerFixture):
+    N = (3, 4, 5, 1, 2, 3, 4, 9, 13, 0)
```

Comment:
you could declare this `N` outside of these functions and share it between the test cases